.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: node_js
+git:
+  depth: false
+
+dist: trusty
+sudo: required
+
+matrix:
+  include:
+    - node_js: 8
+      env: TEST_SUITE=unit
+      install: npm i
+      script: npm run test:unit-ci
+
+    - node_js: 8
+      env: TEST_SUITE=screenshots
+      install: npm i
+      script: npm run test:image-diff-ci
```

Comment:
I suggest you run `./test/screenshot/start.sh && sleep 10s` here before running `npm run test:image-diff`. 

Then you don't even need the test:image-diff-ci NPM command

Reply:
👍